src/ocean/util/log/model/ILogger.d

```diff
@@ -96,52 +129,69 @@ interface ILogger
 
         void fatal (cstring fmt, ...);
 
-        /***********************************************************************
+    /***************************************************************************
```

Comment:
I suppose those are the functions that would be removed later? In that case you can keep it like this I guess :)-